### ISSUE TITLE
has_qsiprep_t1w is not defined

### ISSUE THREAD

**Issue by Shade-** [PennLINC/qsiprep#675]
Hello, I am testing the new 0.20.0 update.

I use FreeSurfer input and use the recon workflow with mrtrix single shell ACT.

While 0.19.1 works like a charm, I am now encountering this as soon as the pipeline loads:

```Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.10/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/usr/local/miniconda/lib/python3.10/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/miniconda/lib/python3.10/site-packages/qsiprep/cli/run.py", line 1198, in build_recon_workflow
    retval['workflow'] = init_qsirecon_wf(
  File "/usr/local/miniconda/lib/python3.10/site-packages/qsiprep/workflows/recon/base.py", line 99, in init_qsirecon_wf
    single_subject_wf = init_single_subject_wf(
  File "/usr/local/miniconda/lib/python3.10/site-packages/qsiprep/workflows/recon/base.py", line 208, in init_single_subject_wf
    anat_ingress_node, available_anatomical_data = init_highres_recon_anatomical_wf(
  File "/usr/local/miniconda/lib/python3.10/site-packages/qsiprep/workflows/recon/anatomical.py", line 162, in init_highres_recon_anatomical_wf
    if has_qsiprep_t1w:
NameError: name 'has_qsiprep_t1w' is not defined
```

I suppose Python 3.10 changed something in global variables declaration.

**Comment by dcoynel** [comment]
I have the same issue. 

I also want to run recon with the "mrtrix_singleshell_ss3t_ACT-hsvs" spec.

**Comment by julfou81** [comment]
Hi there,

Same issue here when testing the reconstruction workflow: `mrtrix_multishell_msmt_ACT-hsvs` with qsiprep singularity image version 0.20.0. 
No issue for the same data with the singularity image of version 0.19.0

**Comment by chiuhoward** [comment]
Saw that @mattcieslak is working on this in #705, but I also just encountered this message when running `mrtrix_singleshell_ss3t_ACT-hsvs`.

**Comment by mattcieslak** [comment]
This is fixed in 0.23.1
